### PR TITLE
Modules Page — fix grammar, spelling, and image style

### PR DESCRIPTION
**Pull request by adrianmcli**
* Fixed spelling and punctuation mistakes.
* Made the link for the demo open in a new tab.
* Added `max-width` setting to image under the Architecture section to prevent it from becoming too large.

### CODE REVIEW

File: guide/module-list/module-list.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ To use Augury, lets open up DevTools with the following shortcut:
 Ctrl + Shift + I (Cmd + Opt + I on Mac)
 ```
 
-When the DevTools panel opens, select the **Augury** tab located on the far right. By default the _Component Tree_ tab will be open. To the far right, there will be a tab labelled _NgModules_. This is the section we want so click on it to open it up!
+When the DevTools panel opens, select the **Augury** tab located on the far right. By default the _Component Tree_ tab will be open. To the far right, there will be a tab labeled _NgModules_. This is the section we want so click on it to open it up!
```
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: Can you also removed the bold from Augury.

File: guide/module-list/module-list.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Feature modules can be thought of as extending the functionality of an applicati
 
 The `SongsModule` also uses the `SongService` as seen under the provider section.
 
-Lets take a look at the definition of the Song Module, open up `src/app/modules/songs.module.ts`:
+Let's take a look at the definition of the Song Module, open up `src/app/modules/songs.module.ts`:
```
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: Change to, "Let us" (style guide no contraction)

File: guide/module-list/module-list.md [hunk 3]
```diff
@@ -145,9 +145,9 @@ export class SongListComponent {
...
+The `SongListComponent` uses the `Logger` service via dependency injection that has been made available from the `AppModule`. Since the `AppModule` is at the root of the application's dependency injection hierarchy, the `SongListComponent` can use any of its providers. However, since the `Logger service` is defined in the `AppModule`, Augury will list this service under the providers section of the `AppModule`. It is not included in the section of the `SongsModule`, even though the `SongsModule` uses this service. To get a better sense of dependency injection throughout our application we should use Augury's Dependency Injection Tree.
 
-Lets see what happens when we remove the `SongService` from the provider list in the `SongModule` definition and move it to the provider list in the `SongsComponent` component definition. Here is what `SongsComponent` will look like:
+Let's see what happens when we remove the `SongService` from the provider list in the `SongModule` definition and move it to the provider list in the `SongsComponent` component definition. Here is what `SongsComponent` will look like:
```
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: Let us

File: guide/module-list/module-list.md [hunk 4]
```diff
@@ -172,7 +172,7 @@ Lets see what happens when we remove the `SongService` from the provider list in
 export class SongsComponent { ...
 ```
 
-Augury has picked up what we've done and has noticed that the `SongService` is not longer a provider from the module level, but instead from the component level. As we can see below, `SongService` is now under the _ProvidersInDeclarations_ section.
+Augury has picked up on what we've done and has noticed that the `SongService` is no longer a provider from the module level, but instead from the component level. As we can see below, `SongService` is now under the _ProvidersInDeclarations_ section.
```
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: I would remove the "on", the original sounded more natural.

File: guide/module-list/module-list.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+We will use a simple Angular application, _Song List_ to examine how to use Augury's module list feature. The application includes a simple form for the user to search for songs and will use the public Spotify API to get the results and display the album art and song title in a list.
 
 ## Architecture
 
-The _Song List_ application is composed of two modules. One, the root application module which is used to bootstrap and launch the application. The second is a single feature module that contains the components, services and pipes relevant for the searching and displaying of song data. Lets take a look at how this is organized:
+The _Song List_ application is composed of two modules. One, the root application module which is used to bootstrap and launch the application. The second is a single feature module that contains the components, services, and pipes relevant for the searching and displaying of song data. Lets take a look at how this is organized:
 
-<img src="images/module-breakdown.png" style="width: 90%" alt="Module Breakdown" />
+<img src="images/module-breakdown.png" style="width: 480px; max-width: 90%;" alt="Module Breakdown" />
```
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: Did you measure the image width here?

Comment (adrianmcli) [inline guide/module-list/module-list.md]: @rajinder-yadav The width of the image is 872px, it looks awkwardly large when you are viewing it on desktop. Take a look: https://augury.angular.io/pages/guides/module-list.html

File: guide/module-list/module-list.md (hunk 1 above)
Comment (adrianmcli) [inline guide/module-list/module-list.md]: Will do.

File: guide/module-list/module-list.md (hunk 2 above)
Comment (adrianmcli) [inline guide/module-list/module-list.md]: Will do.

File: guide/module-list/module-list.md (hunk 3 above)
Comment (adrianmcli) [inline guide/module-list/module-list.md]: Done.

**Comment by adrianmcli** [comment]
@rajinder-yadav just a reminder to take a look at this PR

File: guide/module-list/module-list.md (hunk 5 above)
Comment (rajinder-yadav) [inline guide/module-list/module-list.md]: It was intentional, if someone reduces the size of the page, side by side debugging, presentation, or is on a small viewport it's going to look small. Just leave it as 90%

**Comment by rajinder-yadav** [comment]
I can approve once the image width is changed back, thanks!

**Comment by rajinder-yadav** [comment]
:shipit:

**Comment by adrianmcli** [comment]
@rajinder-yadav I don't have write access, so you'll have to merge it. Thanks!

**Comment by rajinder-yadav** [comment]
Merged.

**Pull request merged**